bms_if.c

```
@@ -421,6 +421,16 @@ static THD_FUNCTION(if_thd, p) {
 }
 
 float bms_if_get_i_in(void) {
+	printf("this is a test");
+	int middle,a,b,c;
+
+    if ((a <= b) && (a <= c)) {
```

cppcheck

[error] Uninitialized variable

(uninitvar)